src/common/transformations/src/transformations/op_conversions/convert_interpolate11_downgrade.cpp

```diff
@@ -42,11 +42,11 @@ ov::pass::ConvertInterpolate11ToInterpolate4::ConvertInterpolate11ToInterpolate4
         if (interpolate_v11->get_attrs().shape_calculation_mode ==
             ov::op::util::InterpolateBase::ShapeCalcMode::SCALES) {
             v4_input_scales = interpolate_v11->input_value(1);
-            v4_input_output_shape = opset4::Constant::create(element::i32, Shape{1}, {1});
+            v4_input_output_shape = opset4::Constant::create(element::i32, Shape{2}, {1});
```

Comment:
`Shape{2}` seems to be hardcoded workaround for a specific case (interpolation over exactly two axes), when the `scales` input shape is expected to be equal to the shape of `sizes` input, no matter of the mode `Interpolate::ShapeCalcMode`.

In general the shape checks of unused input should be relaxed in the source of the issue, 
anyway more universal fix would be usage of `Broadcast` to the `ShapeOf` the other input, instead of magic `Shape{2}`.